omf-website/src/components/code-snippet-test.tsx

```
@@ -0,0 +1,127 @@
+        <Card className="p-4">
+          <h2 className="text-lg font-bold mb-2">Rendered HTML</h2>
+          <pre className="bg-gray-800 text-white p-4 rounded overflow-x-auto">
+            <code>{escapeHtml(renderedHtml[selectedCase] || '')}</code>
+          </pre>
+        </Card>
+        
+        <Card className="p-4 md:col-span-2">
+          <h2 className="text-lg font-bold mb-2">Preview</h2>
+          <div className="bg-gray-800 text-white p-4 rounded overflow-x-auto">
+            <pre>
+              <code dangerouslySetInnerHTML={{ __html: renderedHtml[selectedCase] || '' }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)